handler.py

```diff
@@ -54,48 +57,59 @@ def fxa_validate(event, device_id=None):
     """
     # extract the FxA OAuth token from the Authorization header
     try:
-        auth = event["headers"]["authorization"]
+        assert ("bearer " in event["headers"]["authorization"].lower())
+        auth = event["headers"]["authorization"].split(' ')[1]
```

Comment:
A couple of nits from having had to write similar code too many times in the past:

* `.strip()`ing the header value may be worthwhile, unless your framework already takes care of that for you
* `.startswith('bearer')` is a more precise test than `in`
* You can use `.split(None, 1)` to both split on multiple consecutive whitespace chars, and stop after the first split is found

But all that said, it's hard for things to go too far off the rails here, because if you parse out the wrong thing we'll just reject it on the FxA side.